src/qt/guiutil.cpp

```diff
@@ -941,6 +942,11 @@ QString formatPingTime(double dPingTime)
     return (dPingTime == std::numeric_limits<int64_t>::max()/1e6 || dPingTime == 0) ? QObject::tr("N/A") : QString(QObject::tr("%1 ms")).arg(QString::number((int)(dPingTime * 1000), 10));
 }
 
+QString formatUintValue(uint64_t uValue)
+{
+    return (uValue == std::numeric_limits<int64_t>::max()/1e6 || uValue == 0) ? QObject::tr("N/A") : QString(QObject::tr("%1")).arg(QString::number(uValue), 10);
```

Comment:
Review welcome on the experience here; it'd be nice to turn this into KB, MB, GB, etc... but there could be some Qt feature I don't know of that formats values nicely here.

Reply:
I'd typically write it over a few more lines for simplicity, but... my inclination is there's some really clever ways we could do this, but I'm happy with anything that works.

Reply:
Qt 5.10 introduced [formattedDataSize](https://doc.qt.io/qt-5/qlocale.html#formattedDataSize-1), but I'm not sure we require Qt 5 fully, so I might just write a 10-line loop instead.

Reply:
Recommend to do the 10-line loop for 1.14.4 because technically we still support Qt4

Reply:
Done, but it's not pretty. I could be persuaded to forget the string-handling format code and create the output directly in each if/else branch. What's the least unmaintainable approach here?

Reply:
Personally I like if-else chains, they're not pretty, but everyone know what they do.